lib/routes/app_pages.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:get/get.dart';
+import 'package:pispapp/ui/pages/login.dart';
+import 'package:pispapp/ui/pages/profile.dart';
+import 'package:pispapp/ui/pages/splash.dart';
+
+part './app_routes.dart';
```

Comment:
If you `CTRL+F` in this page for `part`, it is said that the usage of `part` and `part of` is actually discouraged by the Dart team. It is recommended to create a small library instead. `json_serializable` may still depends on it, but I think it is recommended to avoid using that keyword unless it is really necessary.

Reply:
addressed in https://github.com/mojaloop/pisp-demo-app-flutter/pull/12